src/schema/objects/metadata.yaml

```diff
@@ -3347,13 +3457,20 @@ TriggerChannelCount:
     Corresponds to the `TRIG` channel type.
   type: integer
   minimum: 0
-TubingLength:
-  name: TubingLength
-  display_name: Tubing Length
+TrackedPointsCount:
+  name: TrackedPointsCount
+  display_name: Tracked Points Count
   description: |
-    The length of the blood tubing, from the subject to the detector in meters.
+    Number of different tracked points tracked in a motion tracking system.
   type: number
   unit: m
+TrackingSystemName:
+  name: TrackingSystemName
+  display_name: Tracking System Name
+  description: |
+    Name of the tracking system. The value must match the `"tracksys"` label
+    of the corresponding *_motion.tsv file name.
```

Comment:
this would need to be adjusted. We agreed that this field MAY (optionally) be used to record a more complete / human readable name of the `tracksys` entity that it corresponds to. Also needs to be adjusted above in the `tracksys` metadata entry.